pkg/controller/statefulset/stateful_set.go

```diff
@@ -114,28 +110,33 @@ func NewStatefulSetController(podInformer cache.SharedIndexInformer, kubeClient
 			},
 			DeleteFunc: ssc.enqueueStatefulSet,
 		},
+		statefulSetResyncPeriod,
```

Comment:
We should be able to switch to the default resync period but probably not before adding a pvc cache ... Add a TODO